terraform/modules/backend/main.tf

```diff
@@ -0,0 +1,190 @@
+#tfsec:ignore:aws-s3-block-public-policy:block_public_policy is set to true, this is a false flag
+#tfsec:ignore:aws-s3-enable-bucket-encryption:using AES256 encryption with key managed by Amazon S3 instead of KMS
+#tfsec:ignore:aws-s3-ignore-public-acls:ignore_public_acls is set to true, this is a false flag
+#tfsec:ignore:aws-s3-no-public-buckets:restrict_public_buckets is set to true, this is a false flag
+#tfsec:ignore:aws-s3-encryption-customer-key:using key managed by S3 because bucket will only contain access logs for another bucket
+#tfsec:ignore:aws-s3-enable-versioning:versioning not required because log files are unique and only created once
+#tfsec:ignore:aws-s3-enable-bucket-logging:access logs are not required for a log bucket
+#tfsec:ignore:aws-s3-specify-public-access-block:aws_s3_bucket_public_access_block is being used, this is a false flag
+module "tf_state_log_bucket" {
+  #checkov:skip=CKV_TF_1:providing git source with commit hash causes filename too long errors on Checkov, and we provide the version
+  #checkov:skip=CKV_AWS_273:iam user for bucket access isn't required nor created by cloudposse with our configuration, so we don't need to ensure this is controlled through SSO instead
+  #checkov:skip=CKV_AWS_300:lifecycle configuration is set below for aborting failed uploads, looks like a false flag
+  #checkov:skip=CKV2_AWS_34:use of ssm to store a parameter for the access key of an iam user isn't required nor created by cloudposse with our configuration, so we don't need to ensure encryption of the parameter
+  #checkov:skip=CKV2_AWS_62:we don't require event notifications on this bucket as its storing logs continuously and will become a nuisance
+  source = "cloudposse/s3-bucket/aws"
+  # Cloud Posse recommends pinning every module to a specific version
+  version = "3.1.2"
+
+  acl                          = null
+  allow_encrypted_uploads_only = false
+  allow_ssl_requests_only      = false
+  block_public_acls            = true
+  block_public_policy          = true
+  bucket_name                  = var.state_log_bucket_name
+  enabled                      = true
+  environment                  = "meta"
+  force_destroy                = false
+  ignore_public_acls           = true
+  lifecycle_configuration_rules = [{
+    abort_incomplete_multipart_upload_days = 1
+    enabled                                = true
+    expiration = {
+      days                         = 90
+      expired_object_delete_marker = false
+    }
+    filter_and                    = {}
+    id                            = "expire-old-logs"
+    noncurrent_version_expiration = {}
+    noncurrent_version_transition = []
+    transition                    = []
+  }]
+  restrict_public_buckets = true
+  s3_object_ownership     = "BucketOwnerEnforced"
+  s3_replication_enabled  = false
+  source_policy_documents = [data.aws_iam_policy_document.allow_log_writes.json]
+  sse_algorithm           = "AES256"

```

Comment:
We don't think the access logs are that sensitive, so we didn't think it was necessary to use a customer-managed key.